src/routers/products.js

```diff
@@ -0,0 +1,29 @@
+const productsRouter = require('express').Router();
+const { ProductsController } = require('../controllers');
+const {
+  createProductPayloadValidator,
+  updateProductPayloadValidator
+} = require('../middlewares/validators');
+const { verifyRole } = require('../middlewares/auth');
+
+const productsController = new ProductsController();
+productsRouter.post(
+  '/',
+  verifyRole('Admin', 'Super admin'),
```

Comment:
Same as in comment above

Reply:
Fixed. BTW, I couldn't create tests for this.